.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,46 @@
+name: release-workflow
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Release to NPM
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 8
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.17
```

Comment:
It remains 😂

Reply:
I updated it, thanks for your eagle eyes!